### ISSUE TITLE
VoiceOver in iOS does not read questions

### ISSUE THREAD

**Issue by austinseraphin** [UnlockedLab/unlockphilly#54]
Tested with iOS 8.0.2. VoiceOver only reads the radio buttons, no questions.
